Source/ASVideoPlayerNode.mm

```diff
@@ -658,7 +687,10 @@ - (void)removeSpinner
   if (!_spinnerNode) {
     return;
   }
-  [_spinnerNode removeFromSupernode];
+  {
```

Comment:
We may should grab the `_spinnerNode` here before unlocking

Reply:
Let's do this at the end so we don't need to re-acquire the lock.